register-postgres.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "user-access-outbox-connector",
+  "config": {
+    "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+    "tasks.max": "1",
+    "plugin.name": "pgoutput",
+    "database.hostname": "db",
+    "database.port": "5432",
+    "database.user": "postgres",
+    "database.password": "postgres",
+    "database.dbname" : "rails_modular_monolith_with_ddd_development",
```

Comment:
So if this one is used for development, where is this set for production?

And for testing, we won't use any connector, we will just mock things?